### PR TITLE
feat: ability to specify security group name

### PR DESCRIPTION
**Pull request by BScott-c**
When old database instances were NOT created with this terraform module and we migrate it to use this terraform module the security group name is not matching causing further down time in the migration that can be avoided by allowing us to specify the security group name.

### CODE REVIEW

**Review by stuft2 — APPROVED** [review]

**Comment by stuft2** [comment]
I think we have to change all the version numbers in the documentation before we merge

File: variables.tf
Comment (yoshutch) [inline variables.tf:25]: (nit) in most terraform modules I've made this default to `null` and then I would check if `var.db_security_group_name != null` I would use the variable, else the generated default one. I personally don't like setting a default that actually isn't used mainly because it's a little misleading, but again this is more a personal preference than anything.

**Review by yoshutch — CHANGES_REQUESTED** [review]
Yeah, we'll need to bump the terraform version in the README.md to the next minor version since you're just adding a variable, so not breaking change. Then add this new variable to the inputs table. Then I think you're good

**Review by yoshutch — APPROVED** [review]
LGTM as long as you see a valid plan using this. You could even test it out on your other repo by setting the "version" to this branch name like `source = "github.com/byu-oit/terraform-aws-rds?ref=feat/sgName"`

**Comment by BScott-c** [comment]
Couldn't test with the naming of the branch. Use https://github.com/byu-oit/terraform-aws-rds/pull/38 instead

**Pull request closed without merging**
